proto/api.proto

```diff
@@ -52,38 +52,175 @@ service Api {
   };
 
   // Drift V2
+  rpc PostCloseDriftPerpPositions(PostCloseDriftPerpPositionsRequest) returns (PostCloseDriftPerpPositionsResponse) {
+    option (google.api.http) = {
+      post: "/api/v2/drift/perp/close"
+      body: "*"
+    };
+    option (grpc.gateway.protoc_gen_openapiv2.options.openapiv2_operation) = {
+      description: "Returns a partially signed txn signature to close a perp position on Drift";
+      summary: "closes a perp position on Drift";
+      tags: ["Drift V2", "Trade"];
```

Comment:
Trade ->Trade V2. I did the same thing I'm my Openbook V2 changes

Reply:
done